src/Microsoft.ML.TimeSeries/SrCnnAnomalyDetectionBase.cs

```diff
@@ -0,0 +1,302 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.ML.Data;
+using Microsoft.ML.Internal.Utilities;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.Transforms.TimeSeries
+{
+    public class SrCnnAnomalyDetectionBaseWrapper : IStatefulTransformer, ICanSaveModel
```

Comment:
What does `Wrapper` mean for this class?

Also - the file name and the class name should be the same, so it is easy to find files/classes/etc.

Reply:
Changed the class name.